react/AppIcon/index.jsx

```diff
@@ -23,7 +23,12 @@ export class AppIcon extends Component {
       status: preloaded ? DONE : FETCHING
     }
   }
-
+  componentWillMount() {
```

Comment:
Sorry I missed this in a similar PR on drive, but `componentWillMount ` is deprecated and we shouldn't add any. I think setting the flag in the constructor instead is more a good alternative?

Reply:
This! Will use didmount instead. Constructor is not the right place since it’s not related to the life cycle and can already be instanciated